layouts/partials/post.html

```diff
@@ -20,4 +20,10 @@ <h1>{{ .Title }}</h1>
         </p>
     </div>
 </div>
+<aside class="post-toc">
+    <nav id="toc">
+        <ul class="post-toc-content"></ul>
+    </nav>
+</aside>
```

Comment:
Remove extra lines below

Reply:
Lets make this a partial and include it in both post and page

Add if/else attribute to enable